README.md

```diff
@@ -1 +1,271 @@
 # Open Canvas Initial Draft Spec
+
+**NOTE:** This is a draft!
+
+## Overview
+
+- Trying to balance between visual and conceptual canvases. Nodes provide the core elements for the canvas. Relations provide rich support for creating conceptual relationships between nodes.
+- Not trying to maintain backwards-compatibility with Obsidian's JSON Canvas.
+- Aiming for extensibility via schemas.
+- Enable different canvases to work together without canvases needing to implement or even understand all aspects of the spec.
+
+## Structure
+
+The JSON structure includes `nodes`, `relations`, and `schemas`, and a version number that defines the shape of the top-level structure and the shapes of the base classes. The following is a valid, though empty open canvas file:
+
+```json
+{
+  "schema_version": "1.0",
+  "nodes": {},
+  "relations": {},
+  "schemas": {}
+}
+```
+
+### Nodes
+
+All visible items on the canvas are considered Nodes.
+
+**Node Base Class**
+
+All Nodes must implement the following properties:
+
+```ts
+type Node = {
+  id: string;
+  schema: string;
+  schema_version: string;
```

Comment:
Should default to root key `schema_version`.

Reply:
Reasonable.